src/main/java/org/openbase/bco/bcozy/controller/powerTerminal/PowerDrawVisualizationController.java

```diff
@@ -0,0 +1,41 @@
+package org.openbase.bco.bcozy.controller.powerTerminal;
```

Comment:
Please do not break any java naming conventions.

> Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

More Details: [Naming a Package](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html)

Reply:
Package renamed in d58cc33